package.json

```diff
@@ -73,15 +95,17 @@
     "preprocess": "run-p preprocess:*",
     "preprocess:svg-to-twig": "gulp svg-to-twig",
     "preprocess:theo-to-mdx": "gulp theo-to-mdx",
-    "watch-preprocess": "gulp watch",
+    "watch-preprocess": "gulp watch-preprocess",
+    "build": "gulp build",
     "check-lint": "run-s check-lint:*",
-    "check-lint:prettier": "prettier --ignore-path .gitignore --list-different '**/*.{scss,js,mdx}'",
+    "check-lint:prettier": "prettier --ignore-path .gitignore --check '**/*.{scss,js,mdx}'",
```

Comment:
@calebeby Thoughts on the following setup to reduce the duplicate Prettier setup? 🙂 

```js
{
  "prettier": "prettier --ignore-path .gitignore '**/*.{scss,js,mdx}'",
  "check-lint:prettier": "npm run prettier -- --check",
  "lint:prettier": "npm run prettier -- --write"
}
```

Reply:
I changed prettier to pass the directory rather than globs (so it automatically runs on all files it supports), so I think this is no longer necessary.